src/mesos_state_client.erl

```diff
@@ -370,6 +370,11 @@ container_status(_ContainerStatus = #{network_infos := NetworkInfos}) ->
   #container_status{
     cgroup_info = undefined,
     network_infos = network_infos(NetworkInfos, [])
+  };
+container_status(_) ->
```

Comment:
On second thought, it looks like we're already ok with using catch-alls in other parts, so I'm fine with this.